myco_api_client/matching_algorithms/attributed_matching_algorithm.py

```diff
@@ -21,80 +23,95 @@ class AttributedMatchingAlgorithm(BaseMatchingAlgorithm):
     def get_matches_recommendations(
             cls, matching_data: Dict[str, Dict]) -> List[BidOfferMatch.serializable_dict]:
         recommendations = []
-        for market_id, data in matching_data.items():
-            bids_mapping = {bid["id"]: bid for bid in data.get("bids")}
-            offers_mapping = {offer["id"]: offer for offer in data.get("offers")}
+        for market_id, time_slot_data in matching_data.items():
+            for time_slot, data in time_slot_data.items():
+                bids_mapping = {bid["id"]: bid for bid in data.get("bids")}
```

Comment:
This will fail if `data["bids"]` is not present (it can't iterate over `None` values)

Reply:
Same applies below